app/views/practices/_search.js.erb

```diff
@@ -762,13 +762,29 @@ function searchPracticesPage(skipSearchTracking=false) {
 
         appliedFilterButton.className = `usa-button applied-filter applied-filter-${filterClassIndex}`;
         appliedFilterButton.id = `${filterName}`;
-        appliedFilterButton.textContent = appliedFilterString + ' X';
+        appliedFilterButton.textContent = appliedFilterString;
         appliedFilterButton.style.cursor = 'pointer';
         appliedFilterButton.setAttribute(
             'aria-label',
             `Remove ${filterType} filter for ${filterName} and update search results`
         );
 
+        const svgNamespace = "http://www.w3.org/2000/svg";
+        const svgIcon = document.createElementNS(svgNamespace, "svg");
+        svgIcon.setAttribute("xmlns", svgNamespace);
+        svgIcon.setAttribute("viewBox", "0 0 352 512");
+
```

Comment:
add an `aria-hidden="true"` to the <svg> to make sure it doesn't show up in a screenreader's image list:
<img width="806" alt="Screenshot 2024-08-22 at 11 16 59 AM" src="https://github.com/user-attachments/assets/e4e1c065-1ab5-494c-a2fb-4d60b065c66a">